### ISSUE TITLE
Generate is not idempotent for routes

### ISSUE THREAD

**Issue by ariejan** [thoughtbot/administrate#232]
Running `rails generate administrate:install` more than once will add extra admin namespace blocks to `config/routes.rb`. Probably a low priority thing, but people using the generate command to just "reset" their work in progress might run into this.

**Comment by Marthyn** [comment]
Yeah same stuff here, whenever i run a migration i like to update my dashboard automatically too. But i have to remove this every time
```
  namespace :admin do
    DashboardManifest::DASHBOARDS.each do |dashboard_resource|
      resources dashboard_resource
    end

    root controller: DashboardManifest::ROOT_DASHBOARD, action: :index
  end
```

**Comment by graysonwright** [comment]
I think the easiest way to fix this would be checking to see if the routes are defined before generating them again. Thanks for the feedback!

**Comment by graysonwright** [comment]
Fixed by #282.
